manifests/4.2/sriov-network-operator-sriovnetworknodestate.crd.yaml

```diff
@@ -29,6 +29,8 @@ spec:
           type: object
         spec:
           properties:
+            dpConfigVersion:
```

Comment:
I didn't see this is used anywhere `dpConfigVersion`, what should be the value of it?

Reply:
It is not related to this patch, but a update to make sure the crds in `manifests` directory is the same as the ones in `deploy`